From 161225fb99099f67ac4a2b41eb051c062e34a91d Mon Sep 17 00:00:00 2001
From: Oswald Buddenhagen <oswald.buddenhagen@gmx.de>
Date: Fri, 11 Dec 2020 13:12:41 +0100
Subject: [PATCH 16/16] prune bogus comment from linguist_nl

something must have gone wrong during the po => ts conversion.

Change-Id: I84d4b0e7558d371ccba0b69d60ea0c87208d29f9
Reviewed-by: Freek de Kruijf <freekdekruijf@kde.nl>
Reviewed-by: Oswald Buddenhagen <oswald.buddenhagen@gmx.de>
---
 translations/linguist_nl.ts | 1996 +----------------------------------
 1 file changed, 1 insertion(+), 1995 deletions(-)

diff --git a/translations/linguist_nl.ts b/translations/linguist_nl.ts
index a54e224..87eaed9 100644
--- a/translations/linguist_nl.ts
+++ b/translations/linguist_nl.ts
@@ -8,2001 +8,7 @@
     <extra-po-header-po_revision_date>2018-06-15 13:01+0100</extra-po-header-po_revision_date>
     <extra-po-header-project_id_version></extra-po-header-project_id_version>
     <extra-po-header-x_generator>Lokalize 2.0</extra-po-header-x_generator>
-    <extra-po-header_comment># Freek de Kruijf &lt;freek@opensuse.org&gt;, 2014, 2018.
-msgid &quot;&quot;
-msgstr &quot;&quot;
-&quot;Project-Id-Version: \n&quot;
-&quot;PO-Revision-Date: 2018-06-15 13:01+0100\n&quot;
-&quot;Last-Translator: Freek de Kruijf &lt;freekdekruijf@kde.nl&gt;\n&quot;
-&quot;Language-Team: Dutch &lt;kde-i18n-nl@kde.org&gt;\n&quot;
-&quot;Language: nl\n&quot;
-&quot;MIME-Version: 1.0\n&quot;
-&quot;Content-Type: text/plain; charset=UTF-8\n&quot;
-&quot;Content-Transfer-Encoding: 8bit\n&quot;
-&quot;X-Qt-Contexts: true\n&quot;
-&quot;Plural-Forms: nplurals=2; plural=(n != 1);\n&quot;
-&quot;X-Generator: Lokalize 2.0\n&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1363
-msgctxt &quot;AboutDialog|&quot;
-msgid &quot;Qt Linguist&quot;
-msgstr &quot;Qt Linquist&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Qt Linguist - Batch Translation&quot;
-msgstr &quot;Qt Linguist - Vertaling in bulk&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Options&quot;
-msgstr &quot;Opties&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Set translated entries to finished&quot;
-msgstr &quot;Vertaalde items op gereed zetten&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Retranslate entries with existing translation&quot;
-msgstr &quot;Items met bestaande vertalingen opnieuw vertalen&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;&quot;
-&quot;Note that the modified entries will be reset to unfinished if &apos;Set &quot;
-&quot;translated entries to finished&apos; above is unchecked&quot;
-msgstr &quot;&quot;
-&quot;Merk op dat de gewijzigde items gereset zullen worden naar niet-gereed als &quot;
-&quot;bovenstaand &apos;Vertaalde items op gereed zetten&apos; niet is gedeactiveerd&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Translate also finished entries&quot;
-msgstr &quot;Ook items gereed vertalen&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Phrase book preference&quot;
-msgstr &quot;Voorkeur voor uitdrukkingenboek&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Move up&quot;
-msgstr &quot;Omhoog&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Move down&quot;
-msgstr &quot;Omlaag&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;&quot;
-&quot;The batch translator will search through the selected phrase books in the &quot;
-&quot;order given above&quot;
-msgstr &quot;&quot;
-&quot;De bulkvertaler zal de geselecteerde uitdrukkingenboeken doorzoeken in de &quot;
-&quot;boven gegeven volgorde&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;&amp;Run&quot;
-msgstr &quot;Uitvoe&amp;ren&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslation.ui:-1
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Cancel&quot;
-msgstr &quot;Annuleren&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslationdialog.cpp:66
-#, qt-format
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Batch Translation of &apos;%1&apos; - Qt Linguist&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslationdialog.cpp:103
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Searching, please wait...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslationdialog.cpp:103
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;&amp;Cancel&quot;
-msgstr &quot;&amp;Annuleren&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslationdialog.cpp:145
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Linguist batch translator&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/batchtranslationdialog.cpp:146
-#, qt-format
-msgctxt &quot;BatchTranslationDialog|&quot;
-msgid &quot;Batch translated %n entries&quot;
-msgid_plural &quot;Batch translated %n entries&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:200
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;The translation file &apos;%1&apos; will not be loaded because it is empty.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:207
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;&lt;qt&gt;Duplicate messages found in &apos;%1&apos;:&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:211
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:219
-msgctxt &quot;DataModel|&quot;
-msgid &quot;&lt;p&gt;[more duplicates omitted]&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:214
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;&lt;p&gt;* ID: %1&quot;
-msgstr &quot;&lt;p&gt;* ID: %1&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:222
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;&lt;p&gt;* Context: %1&lt;br&gt;* Source: %2&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:225
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;&lt;br&gt;* Comment: %3&quot;
-msgstr &quot;&lt;br&gt;* Toelichting: %3&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:295
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;&quot;
-&quot;Linguist does not know the plural rules for &apos;%1&apos;.\n&quot;
-&quot;Will assume a single universal form.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:351
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;Cannot create &apos;%2&apos;: %1&quot;
-msgstr &quot;Kan &apos;%2&apos; niet aanmaken: %1&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:401
-#, qt-format
-msgctxt &quot;DataModel|&quot;
-msgid &quot;%1 (%2)&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:411
-msgctxt &quot;DataModel|&quot;
-msgid &quot;Universal Form&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:62
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;Accelerator possibly superfluous in translation.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:65
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;Accelerator possibly missing in translation.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:68
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;&quot;
-&quot;Translation does not have same leading and trailing whitespace as the source &quot;
-&quot;text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:71
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;Translation does not end with the same punctuation as the source text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:74
-#, qt-format
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;A phrase book suggestion for &apos;%1&apos; was ignored.&quot;
-msgstr &quot;Een suggestie uit het uitdrukkingenboek voor &apos;%1&apos; is genegeerd.&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:77
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;&quot;
-&quot;Translation does not refer to the same place markers as in the source text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:80
-#, qt-format
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;Translation does not contain the necessary %n/%Ln place marker.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/errorsview.cpp:83
-msgctxt &quot;ErrorsView|&quot;
-msgid &quot;Unknown error&quot;
-msgstr &quot;Onbekende fout&quot;
-
-#: ../../qttools/src/linguist/shared/po.cpp:887
-msgctxt &quot;FMT|&quot;
-msgid &quot;GNU Gettext localization files&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/shared/po.cpp:894
-msgctxt &quot;FMT|&quot;
-msgid &quot;GNU Gettext localization template files&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/shared/qm.cpp:724
-msgctxt &quot;FMT|&quot;
-msgid &quot;Compiled Qt translations&quot;
-msgstr &quot;Gecompileerde Qt-vertalingen&quot;
-
-#: ../../qttools/src/linguist/shared/qph.cpp:180
-msgctxt &quot;FMT|&quot;
-msgid &quot;Qt Linguist &apos;Phrase Book&apos;&quot;
-msgstr &quot;Qt Linguist &apos;Uitdrukkingenboek&apos;&quot;
-
-#: ../../qttools/src/linguist/shared/ts.cpp:689
-msgctxt &quot;FMT|&quot;
-msgid &quot;Qt translation sources&quot;
-msgstr &quot;Qt vertalingsbronnen&quot;
-
-#: ../../qttools/src/linguist/shared/xliff.cpp:837
-msgctxt &quot;FMT|&quot;
-msgid &quot;XLIFF localization files&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Find&quot;
-msgstr &quot;Zoeken&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;&quot;
-&quot;This window allows you to search for some text in the translation source &quot;
-&quot;file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;&amp;Find what:&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Type in the text to search for.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Options&quot;
-msgstr &quot;Opties&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Lets you use a Perl-compatible regular expression&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Regular &amp;expression&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Texts such as &apos;TeX&apos; and &apos;tex&apos; are considered as different when checked.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;&amp;Match case&quot;
-msgstr &quot;&amp;Hoofdlettergevoelig&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Source texts are searched when checked.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;&amp;Source texts&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Obsoleted messages are skipped when checked.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Skip &amp;obsolete&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Comments and contexts are searched when checked.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;&amp;Comments&quot;
-msgstr &quot;&amp;Commentaar&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Ignore &amp;accelerators&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Translations are searched when checked.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;&amp;Translations&quot;
-msgstr &quot;Ver&amp;talingen&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Click here to find the next occurrence of the text you typed in.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Find Next&quot;
-msgstr &quot;Volgende zoeken&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Click here to close this window.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/finddialog.ui:-1
-msgctxt &quot;FindDialog|&quot;
-msgid &quot;Cancel&quot;
-msgstr &quot;Annuleren&quot;
-
-# Freek de Kruijf &lt;freek@opensuse.org&gt;, 2014, 2018.
-#: ../../qttools/src/linguist/linguist/finddialog.cpp:29
-#, fuzzy
-#| msgid &quot;&quot;
-msgctxt &quot;&quot;
-&quot;FindDialog|Choose Edit~|Find from the menu bar or press Ctrl+F to pop up the &quot;
-&quot;Find dialog&quot;
-msgid &quot;&quot;
-msgstr &quot;&quot;
-&quot;Project-Id-Version: \n&quot;
-&quot;PO-Revision-Date: 2018-06-15 13:01+0100\n&quot;
-&quot;Last-Translator: Freek de Kruijf &lt;freekdekruijf@kde.nl&gt;\n&quot;
-&quot;Language-Team: Dutch &lt;kde-i18n-nl@kde.org&gt;\n&quot;
-&quot;Language: nl\n&quot;
-&quot;MIME-Version: 1.0\n&quot;
-&quot;Content-Type: text/plain; charset=UTF-8\n&quot;
-&quot;Content-Transfer-Encoding: 8bit\n&quot;
-&quot;X-Qt-Contexts: true\n&quot;
-&quot;Plural-Forms: nplurals=2; plural=(n != 1);\n&quot;
-&quot;X-Generator: Lokalize 2.0\n&quot;
-
-#. translate, but don&apos;t change
-#: ../../qttools/src/linguist/linguist/messageeditorwidgets.cpp:301
-msgctxt &quot;FormMultiWidget|&quot;
-msgid &quot;Alt+Delete&quot;
-msgstr &quot;Alt+Delete&quot;
-
-#. translate, but don&apos;t change
-#: ../../qttools/src/linguist/linguist/messageeditorwidgets.cpp:302
-msgctxt &quot;FormMultiWidget|&quot;
-msgid &quot;Shift+Alt+Insert&quot;
-msgstr &quot;&quot;
-
-#. translate, but don&apos;t change
-#: ../../qttools/src/linguist/linguist/messageeditorwidgets.cpp:303
-msgctxt &quot;FormMultiWidget|&quot;
-msgid &quot;Alt+Insert&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditorwidgets.cpp:469
-msgctxt &quot;FormMultiWidget|&quot;
-msgid &quot;Confirmation - Qt Linguist&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditorwidgets.cpp:470
-msgctxt &quot;FormMultiWidget|&quot;
-msgid &quot;Delete non-empty length variant?&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/shared/qm.cpp:695
-#, qt-format
-msgctxt &quot;LRelease|&quot;
-msgid &quot;Dropped %n message(s) which had no ID.&quot;
-msgid_plural &quot;Dropped %n message(s) which had no ID.&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/shared/qm.cpp:699
-#, qt-format
-msgctxt &quot;LRelease|&quot;
-msgid &quot;Excess context/disambiguation dropped from %n message(s).&quot;
-msgid_plural &quot;Excess context/disambiguation dropped from %n message(s).&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/shared/qm.cpp:708
-#, qt-format
-msgctxt &quot;LRelease|&quot;
-msgid &quot;    Generated %n translation(s) (%1 finished and %2 unfinished)&quot;
-msgid_plural &quot;    Generated %n translation(s) (%1 finished and %2 unfinished)&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/shared/qm.cpp:712
-#, qt-format
-msgctxt &quot;LRelease|&quot;
-msgid &quot;    Ignored %n untranslated source text(s)&quot;
-msgid_plural &quot;    Ignored %n untranslated source text(s)&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;MainWindow&quot;
-msgstr &quot;Hoofdvenster&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Phrases&quot;
-msgstr &quot;&amp;Uitdrukkingen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Close Phrase Book&quot;
-msgstr &quot;Uitdrukkingenboek &amp;sluiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Edit Phrase Book&quot;
-msgstr &quot;Uitdrukkingenboek b&amp;ewerken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Print Phrase Book&quot;
-msgstr &quot;Uitdrukkingenboek af&amp;drukken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;V&amp;alidation&quot;
-msgstr &quot;V&amp;alidatie&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;View&quot;
-msgstr &quot;&amp;Beeld&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Vie&amp;ws&quot;
-msgstr &quot;Weergave&amp;n&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Toolbars&quot;
-msgstr &quot;&amp;Werkbalken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Zoom&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Guesses&quot;
-msgstr &quot;Geraden&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Help&quot;
-msgstr &quot;&amp;Help&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Translation&quot;
-msgstr &quot;Ver&amp;taling&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;File&quot;
-msgstr &quot;&amp;Bestand&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Recently Opened &amp;Files&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Edit&quot;
-msgstr &quot;Be&amp;werken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Open...&quot;
-msgstr &quot;&amp;Openen...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Open a Qt translation source file (TS file) for editing&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+O&quot;
-msgstr &quot;Ctrl+O&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;E&amp;xit&quot;
-msgstr &quot;&amp;Afsluiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Close this window and exit.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+Q&quot;
-msgstr &quot;Ctrl+Q&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save&quot;
-msgstr &quot;Opslaan&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save changes made to this Qt translation source file&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2068
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2079
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save &amp;As...&quot;
-msgstr &quot;Opslaan &amp;als...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save As...&quot;
-msgstr &quot;Opslaan als...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save changes made to this Qt translation source file into a new file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:854
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2069
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Release&quot;
-msgstr &quot;Uitgave&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Create a Qt message file suitable for released applications from the current &quot;
-&quot;message file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Print...&quot;
-msgstr &quot;Af&amp;drukken...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Print a list of all the translation units in the current translation source &quot;
-&quot;file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+P&quot;
-msgstr &quot;Ctrl+P&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Undo&quot;
-msgstr &quot;&amp;Ongedaan maken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Undo the last editing operation performed on the current translation.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+Z&quot;
-msgstr &quot;Ctrl+Z&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Redo&quot;
-msgstr &quot;Opnie&amp;uw&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Redo an undone editing operation performed on the translation.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+Y&quot;
-msgstr &quot;Ctrl+Y&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Cu&amp;t&quot;
-msgstr &quot;Kni&amp;ppen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Copy the selected translation text to the clipboard and deletes it.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+X&quot;
-msgstr &quot;Ctrl+X&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Copy&quot;
-msgstr &quot;&amp;Kopiëren&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Copy the selected translation text to the clipboard.&quot;
-msgstr &quot;De geselecteerde tekst van de vertaling naar het klembord kopiëren.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+C&quot;
-msgstr &quot;Ctrl+C&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Paste&quot;
-msgstr &quot;P&amp;lakken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Paste the clipboard text into the translation.&quot;
-msgstr &quot;Plak de tekst van het klembord in de vertaling.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+V&quot;
-msgstr &quot;Ctrl+V&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Select &amp;All&quot;
-msgstr &quot;&amp;Alles selecteren&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Select the whole translation text.&quot;
-msgstr &quot;De gehele te vertalen tekst selecteren.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+A&quot;
-msgstr &quot;Ctrl+A&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Find...&quot;
-msgstr &quot;&amp;Zoeken...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Search for some text in the translation source file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+F&quot;
-msgstr &quot;Ctrl+F&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Find &amp;Next&quot;
-msgstr &quot;Vol&amp;gende zoeken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Continue the search where it was left.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;F3&quot;
-msgstr &quot;F3&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Prev Unfinished&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Previous unfinished item&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Move to the previous unfinished item.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+K&quot;
-msgstr &quot;Ctrl+K&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Next Unfinished&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Next unfinished item&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Move to the next unfinished item.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+J&quot;
-msgstr &quot;Ctrl+J&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;P&amp;rev&quot;
-msgstr &quot;Vo&amp;rige&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Move to previous item&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Move to the previous item.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+Shift+K&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ne&amp;xt&quot;
-msgstr &quot;Vol&amp;gende&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Next item&quot;
-msgstr &quot;Volgende item&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Move to the next item.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+Shift+J&quot;
-msgstr &quot;Ctrl+Shift+J&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Done and Next&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Mark item as done and move to the next unfinished item&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Mark this item as done and move to the next unfinished item.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Copy from source text&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Copies the source text into the translation field&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Copies the source text into the translation field.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+B&quot;
-msgstr &quot;Ctrl+B&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Accelerators&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Toggles the validity check of accelerators&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Toggles the validity check of accelerators, i.e. whether the number of &quot;
-&quot;ampersands in the source and translation text is the same. If the check &quot;
-&quot;fails, a message is shown in the warnings window.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Surrounding &amp;Whitespace&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Toggles the validity check of surrounding whitespace.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Toggles the validity check of surrounding whitespace. If the check fails, a &quot;
-&quot;message is shown in the warnings window.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Ending Punctuation&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Toggles the validity check of ending punctuation&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Toggles the validity check of ending punctuation. If the check fails, a &quot;
-&quot;message is shown in the warnings window.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Phrase matches&quot;
-msgstr &quot;Overeenkomsten met &amp;uitdrukkingen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Toggles checking that phrase suggestions are used&quot;
-msgstr &quot;Schakelt controle om dat suggesties voor uitdrukkingen worden gebruikt&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Toggles checking that phrase suggestions are used. If the check fails, a &quot;
-&quot;message is shown in the warnings window.&quot;
-msgstr &quot;&quot;
-&quot;Schakelt controle om dat suggesties voor uitdrukkingen worden gebruikt. Als &quot;
-&quot;de controle mislukt, dan wordt een bericht getoond in het &quot;
-&quot;waarschuwingsvenster.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Place &amp;Marker Matches&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Toggles the validity check of place markers&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Toggles the validity check of place markers, i.e. whether %1, %2, ... are &quot;
-&quot;used consistently in the source text and translation text. If the check &quot;
-&quot;fails, a message is shown in the warnings window.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;New Phrase Book...&quot;
-msgstr &quot;&amp;Nieuw uitdrukkingenboek...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Create a new phrase book.&quot;
-msgstr &quot;Een nieuw uitdrukkingenboek aanmaken.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+N&quot;
-msgstr &quot;Ctrl+N&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Open Phrase Book...&quot;
-msgstr &quot;Uitdrukkingenboek &amp;openen...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Open a phrase book to assist translation.&quot;
-msgstr &quot;Een uitdrukkingenboek openen om te assisteren bij vertaling.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+H&quot;
-msgstr &quot;Ctrl+H&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Reset Sorting&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Sort the items back in the same order as in the message file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Display guesses&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Set whether or not to display translation guesses.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Statistics&quot;
-msgstr &quot;&amp;Statistieken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Display translation statistics.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Manual&quot;
-msgstr &quot;Hand&amp;matig&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;F1&quot;
-msgstr &quot;F1&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;About Qt Linguist&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;About Qt&quot;
-msgstr &quot;Info over Qt&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Display information about the Qt toolkit by Digia.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;What&apos;s This?&quot;
-msgstr &quot;&amp;Wat is dit?&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;What&apos;s This?&quot;
-msgstr &quot;Wat is dit?&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Enter What&apos;s This? mode.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Shift+F1&quot;
-msgstr &quot;Shift+F1&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Search And Translate...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Replace the translation on all entries that matches the search source text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2104
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Batch Translation...&quot;
-msgstr &quot;&amp;Bulk vertaling...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Batch translate all entries using the information in the phrase books.&quot;
-msgstr &quot;&quot;
-&quot;Vertaal alle items in bulk met gebruik van de informatie in de &quot;
-&quot;uitdrukkingenboeken.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2070
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2080
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Release As...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Create a Qt message file suitable for released applications from the current &quot;
-&quot;message file. The filename will automatically be determined from the name of &quot;
-&quot;the TS file.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2156
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;File&quot;
-msgstr &quot;Bestand&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2163
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Edit&quot;
-msgstr &quot;Bewerken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2169
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Translation&quot;
-msgstr &quot;Vertaling&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2175
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Validation&quot;
-msgstr &quot;Validatie&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2182
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Help&quot;
-msgstr &quot;Help&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Open/Refresh Form &amp;Preview&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Form Preview Tool&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;F5&quot;
-msgstr &quot;F5&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2103
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Translation File &amp;Settings...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Add to Phrase Book&quot;
-msgstr &quot;&amp;Aan uitdrukkingenboek toevoegen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+T&quot;
-msgstr &quot;Ctrl+T&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Open Read-O&amp;nly...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Save All&quot;
-msgstr &quot;Alles op&amp;slaan&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+S&quot;
-msgstr &quot;Ctrl+S&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2075
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Release All&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Close&quot;
-msgstr &quot;Sluiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Close All&quot;
-msgstr &quot;Alles &amp;sluiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+W&quot;
-msgstr &quot;Ctrl+W&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Length Variants&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Visualize whitespace&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Toggles visualize whitespace in editors&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Increase&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl++&quot;
-msgstr &quot;Ctrl++&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Decrease&quot;
-msgstr &quot;Verminderen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+-&quot;
-msgstr &quot;Ctrl+-&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Reset to default&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+0&quot;
-msgstr &quot;Ctrl+0&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Show more&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Alt++&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Show fewer&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Alt+-&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Alt+0&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;D&amp;one&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Mark item as done&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.ui:-1
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Mark this item as done.&quot;
-msgstr &quot;&quot;
-
-# Freek de Kruijf &lt;freek@opensuse.org&gt;, 2014, 2018.
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:29
-#, fuzzy
-#| msgid &quot;&quot;
-msgctxt &quot;MainWindow|This is the application&apos;s main window.&quot;
-msgid &quot;&quot;
-msgstr &quot;&quot;
-&quot;Project-Id-Version: \n&quot;
-&quot;PO-Revision-Date: 2018-06-15 13:01+0100\n&quot;
-&quot;Last-Translator: Freek de Kruijf &lt;freekdekruijf@kde.nl&gt;\n&quot;
-&quot;Language-Team: Dutch &lt;kde-i18n-nl@kde.org&gt;\n&quot;
-&quot;Language: nl\n&quot;
-&quot;MIME-Version: 1.0\n&quot;
-&quot;Content-Type: text/plain; charset=UTF-8\n&quot;
-&quot;Content-Transfer-Encoding: 8bit\n&quot;
-&quot;X-Qt-Contexts: true\n&quot;
-&quot;Plural-Forms: nplurals=2; plural=(n != 1);\n&quot;
-&quot;X-Generator: Lokalize 2.0\n&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:215
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Source text&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:216
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:241
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Index&quot;
-msgstr &quot;Index&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:239
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:303
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Context&quot;
-msgstr &quot;Context&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:240
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Items&quot;
-msgstr &quot;Items&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:320
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;This panel lists the source contexts.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:333
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Strings&quot;
-msgstr &quot;Tekenreeksen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:370
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Phrases and guesses&quot;
-msgstr &quot;Uitdrukkingen en probeersels&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:380
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Sources and Forms&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:395
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Warnings&quot;
-msgstr &quot;Waarschuwingen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:454
-msgctxt &quot;MainWindow|status bar: file(s) modified&quot;
-msgid &quot; MOD &quot;
-msgstr &quot; MOD &quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:594
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Loading...&quot;
-msgstr &quot;Bezig met laden...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:626
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:648
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Loading File - Qt Linguist&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:627
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;The file &apos;%1&apos; does not seem to be related to the currently open file(s) &quot;
-&quot;&apos;%2&apos;.\n&quot;
-&quot;\n&quot;
-&quot;Close the open file(s) first?&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:649
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;The file &apos;%1&apos; does not seem to be related to the file &apos;%2&apos; which is being &quot;
-&quot;loaded as well.\n&quot;
-&quot;\n&quot;
-&quot;Skip loading the first named file?&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:710
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;%n translation unit(s) loaded.&quot;
-msgid_plural &quot;%n translation unit(s) loaded.&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:794
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Related files (%1);;&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:798
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Open Translation Files&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:808
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:839
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;File saved.&quot;
-msgstr &quot;Bestand opgeslagen.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:855
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Qt message files for released applications (*.qm)\n&quot;
-&quot;All files (*)&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:858
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:870
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;File created.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:904
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1252
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Printing...&quot;
-msgstr &quot;Bezig met afdrukken...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:912
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Context: %1&quot;
-msgstr &quot;Context: %1&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:944
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;finished&quot;
-msgstr &quot;voltooid&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:947
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;unresolved&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:951
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;obsolete&quot;
-msgstr &quot;verouderd&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:966
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1265
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Printing... (page %1)&quot;
-msgstr &quot;Bezig met afdrukken... (pagina %1)&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:973
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1272
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Printing completed&quot;
-msgstr &quot;Afdrukken voltooid&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:975
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1274
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Printing aborted&quot;
-msgstr &quot;Afdrukken afgebroken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1043
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Search wrapped.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1060
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1337
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1370
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1394
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1416
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1969
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1970
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2267
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2307
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2317
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Qt Linguist&quot;
-msgstr &quot;Qt Linquist&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1061
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1162
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Cannot find the string &apos;%1&apos;.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1080
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Search And Translate in &apos;%1&apos; - Qt Linguist&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1114
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1137
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1161
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Translate - Qt Linguist&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1115
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Translated %n entry(s)&quot;
-msgid_plural &quot;Translated %n entry(s)&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1138
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;No more occurrences of &apos;%1&apos;. Start over?&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1168
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Create New Phrase Book&quot;
-msgstr &quot;Een nieuw uitdrukkingenboek aanmaken&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1169
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;Qt phrase books (*.qph)\n&quot;
-&quot;All files (*)&quot;
-msgstr &quot;&quot;
-&quot;Qt-uitdrukkingenboeken (*.qph)\n&quot;
-&quot;Alle bestanden (*)&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1180
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Phrase book created.&quot;
-msgstr &quot;Uitdrukkingenboek aangemaakt.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1197
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Open Phrase Book&quot;
-msgstr &quot;Uitdrukkingenboek openen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1198
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Qt phrase books (*.qph);;All files (*)&quot;
-msgstr &quot;Qt-uitdrukkingenboeken (*.qph);;Alle bestanden (*)&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1205
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;%n phrase(s) loaded.&quot;
-msgid_plural &quot;%n phrase(s) loaded.&quot;
-msgstr[0] &quot;%n uitdrukking(en) geladen.&quot;
-msgstr[1] &quot;%n uitdrukking(en) geladen.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1298
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1301
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1308
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Add to phrase book&quot;
-msgstr &quot;Aan uitdrukkingenboek toevoegen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1299
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;No appropriate phrasebook found.&quot;
-msgstr &quot;Geen toepasselijk uitdrukkingenboek gevonden.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1302
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Adding entry to phrasebook %1&quot;
-msgstr &quot;Een item toevoegen aan uitdrukkingenboek %1&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1309
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Select phrase book to add to&quot;
-msgstr &quot;Uitdrukkingenboek selecteren om aan toe te voegen&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1338
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Unable to launch Qt Assistant (%1)&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1354
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Version %1&quot;
-msgstr &quot;Versie %1&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1357
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&quot;
-&quot;&lt;center&gt;&lt;img src=\&quot;:/images/splash.png\&quot;/&gt;&lt;/img&gt;&lt;p&gt;%1&lt;/p&gt;&lt;/center&gt;&lt;p&gt;Qt &quot;
-&quot;Linguist is a tool for adding translations to Qt applications.&lt;/&quot;
-&quot;p&gt;&lt;p&gt;Copyright (C) %2 The Qt Company Ltd.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1395
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Do you want to save the modified files?&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1417
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Do you want to save &apos;%1&apos;?&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1461
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Qt Linguist[*]&quot;
-msgstr &quot;Qt Linguist[*]&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1463
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;%1[*] - Qt Linguist&quot;
-msgstr &quot;%1[*] - Qt Linguist&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1720
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1732
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;No untranslated translation units left.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1954
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Window&quot;
-msgstr &quot;&amp;Venster&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1956
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Minimize&quot;
-msgstr &quot;Minimaliseren&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1957
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Ctrl+M&quot;
-msgstr &quot;Ctrl+M&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1969
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Display the manual for %1.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:1970
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Display information about %1.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2061
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Save &apos;%1&apos;&quot;
-msgstr &quot;&apos;%1&apos; op&amp;slaan&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2062
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save &apos;%1&apos; &amp;As...&quot;
-msgstr &quot;&apos;%1&apos; opslaan &amp;als...&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2063
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Release &apos;%1&apos;&quot;
-msgstr &quot;&apos;%1&apos; vrijgeven&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2064
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Release &apos;%1&apos; As...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2065
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Close &apos;%1&apos;&quot;
-msgstr &quot;&apos;%1&apos; sl&amp;uiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2067
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2082
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Save&quot;
-msgstr &quot;Op&amp;slaan&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2071
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2084
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Close&quot;
-msgstr &quot;Sl&amp;uiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2074
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Save All&quot;
-msgstr &quot;Alles opslaan&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2076
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Close All&quot;
-msgstr &quot;Alles sluiten&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2083
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Release&quot;
-msgstr &quot;&amp;Uitgave&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2099
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Translation File &amp;Settings for &apos;%1&apos;...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2100
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;&amp;Batch Translation of &apos;%1&apos;...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2101
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Search And &amp;Translate in &apos;%1&apos;...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2105
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Search And &amp;Translate...&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2268
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Cannot read from phrase book &apos;%1&apos;.&quot;
-msgstr &quot;Kan uit uitdrukkingenboek &apos;%1&apos; niet lezen.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2283
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Close this phrase book.&quot;
-msgstr &quot;Dit uitdrukkingenboek sluiten.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2287
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Enables you to add, modify, or delete entries in this phrase book.&quot;
-msgstr &quot;&quot;
-&quot;Stelt u in staat om items toe te voegen, te wijzigen of te verwijderen in &quot;
-&quot;dit uitdrukkingenboek.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2292
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Print the entries in this phrase book.&quot;
-msgstr &quot;De items in dit uitdrukkingenboek afdrukken.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2308
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Cannot create phrase book &apos;%1&apos;.&quot;
-msgstr &quot;Kan uitdrukkingenboek &apos;%1&apos; niet aanmaken.&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2318
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;Do you want to save phrase book &apos;%1&apos;?&quot;
-msgstr &quot;Wilt u uitdrukkingenboek &apos;%1&apos; opslaan?&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2352
-#, qt-format
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;%n unfinished message(s) left.&quot;
-msgid_plural &quot;%n unfinished message(s) left.&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/mainwindow.cpp:2700
-msgctxt &quot;MainWindow|&quot;
-msgid &quot;All&quot;
-msgstr &quot;Alles&quot;
-
-# Freek de Kruijf &lt;freek@opensuse.org&gt;, 2014, 2018.
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:29
-#, fuzzy
-#| msgid &quot;&quot;
-msgctxt &quot;MessageEditor|This is the right panel of the main window.&quot;
-msgid &quot;&quot;
-msgstr &quot;&quot;
-&quot;Project-Id-Version: \n&quot;
-&quot;PO-Revision-Date: 2018-06-15 13:01+0100\n&quot;
-&quot;Last-Translator: Freek de Kruijf &lt;freekdekruijf@kde.nl&gt;\n&quot;
-&quot;Language-Team: Dutch &lt;kde-i18n-nl@kde.org&gt;\n&quot;
-&quot;Language: nl\n&quot;
-&quot;MIME-Version: 1.0\n&quot;
-&quot;Content-Type: text/plain; charset=UTF-8\n&quot;
-&quot;Content-Transfer-Encoding: 8bit\n&quot;
-&quot;X-Qt-Contexts: true\n&quot;
-&quot;Plural-Forms: nplurals=2; plural=(n != 1);\n&quot;
-&quot;X-Generator: Lokalize 2.0\n&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:104
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;&quot;
-&quot;This whole panel allows you to view and edit the translation of some source &quot;
-&quot;text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:114
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;Source text&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:116
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;This area shows the source text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:120
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;Source text (Plural)&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:122
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;This area shows the plural form of the source text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:126
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;Developer comments&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:129
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;&quot;
-&quot;This area shows a comment that may guide you, and the context in which the &quot;
-&quot;text occurs.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:190
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;&quot;
-&quot;Here you can enter comments for your own use. They have no effect on the &quot;
-&quot;translated applications.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:451
-#, qt-format
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;Translation to %1 (%2)&quot;
-msgstr &quot;Vertaling naar %1 (%2)&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:460
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;&quot;
-&quot;This is where you can enter or modify the translation of the above source &quot;
-&quot;text.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:465
-#, qt-format
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;Translation to %1&quot;
-msgstr &quot;Vertaling naar %1&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:466
-#, qt-format
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;Translator comments for %1&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messageeditor.cpp:590
-#, qt-format
-msgctxt &quot;MessageEditor|&quot;
-msgid &quot;&quot;
-&quot;&apos;%1&apos;\n&quot;
-&quot;Line: %2&quot;
-msgstr &quot;&quot;
-&quot;&apos;%1&apos;\n&quot;
-&quot;Regel: %2&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:1271
-#, qt-format
-msgctxt &quot;MessageModel|&quot;
-msgid &quot;Completion status for %1&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:1286
-msgctxt &quot;MessageModel|&quot;
-msgid &quot;&lt;file header&gt;&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:1288
-msgctxt &quot;MessageModel|&quot;
-msgid &quot;&lt;context comment&gt;&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:1359
-msgctxt &quot;MessageModel|&quot;
-msgid &quot;&lt;unnamed context&gt;&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/messagemodel.cpp:1365
-#, qt-format
-msgctxt &quot;MessageModel|&quot;
-msgid &quot;%n unfinished message(s) left.&quot;
-msgid_plural &quot;%n unfinished message(s) left.&quot;
-msgstr[0] &quot;&quot;
-msgstr[1] &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrase.cpp:178
-#, qt-format
-msgctxt &quot;PhraseBook|&quot;
-msgid &quot;Parse error at line %1, column %2 (%3).&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Edit Phrase Book&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&quot;
-&quot;This window allows you to add, modify, or delete entries in a phrase book.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&amp;Translation:&quot;
-msgstr &quot;Ver&amp;taling:&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&quot;
-&quot;This is the phrase in the target language corresponding to the source phrase.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;S&amp;ource phrase:&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;This is a definition for the source phrase.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;This is the phrase in the source language.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&amp;Definition:&quot;
-msgstr &quot;&amp;Definitie:&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Click here to add the phrase to the phrase book.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&amp;New Entry&quot;
-msgstr &quot;&amp;Nieuw item&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Click here to remove the entry from the phrase book.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&amp;Remove Entry&quot;
-msgstr &quot;Item ve&amp;rwijderen&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Settin&amp;gs...&quot;
-msgstr &quot;Instellin&amp;gen...&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Click here to save the changes made.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;&amp;Save&quot;
-msgstr &quot;Op&amp;slaan&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Click here to close this window.&quot;
-msgstr &quot;&quot;
-
-#: ../../qttools/src/linguist/linguist/phrasebookbox.ui:-1
-msgctxt &quot;PhraseBookBox|&quot;
-msgid &quot;Close&quot;
-msgstr &quot;Sluiten&quot;
-
-# Freek de Kruijf &lt;freek@opensuse.org&gt;, 2014, 2018.
-#: ../../qttools/src/linguist/linguist/phrasebookbox.cpp:29
-#, fuzzy
-#| msgid &quot;&quot;</extra-po-header_comment>
+    <extra-po-header_comment># Freek de Kruijf &lt;freek@opensuse.org&gt;, 2014, 2018.</extra-po-header_comment>
     <extra-po-headers>Project-Id-Version,PO-Revision-Date,Last-Translator,Language-Team,Language,MIME-Version,Content-Type,Content-Transfer-Encoding,X-Qt-Contexts,Plural-Forms,X-Generator</extra-po-headers>
 <context>
     <name>AboutDialog</name>
-- 
2.31.1

